host/3.0/buster/amd64/base/out/java/java11-appservice.Dockerfile

```diff
@@ -0,0 +1,73 @@
+ARG JAVA_VERSION=11u7
+
+ARG HOST_VERSION=3.2.0
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host.git /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
```

Comment:
no need to build host again here?

Reply:
This is the out directory. Maybe it should be in the root of the repo so it's clearer? i.e:

```
out/host/3.0/amd64/java/java11-appservice.Docker
```

Reply:
Thank you for the suggestion Ahmed.  I will probably use that path when I am finished implementing this across versions. My plan is to bump it up to all languages v3 -appservice.  Then see what I can do about the other images we have in v3. Then update it for v4. 
I wrote this down and will tag you when I have a PR that incorporates it.